integration-tests/src/test/resources/onpremdomain/config/config.xml

```diff
@@ -0,0 +1,168 @@
+    <node-manager-username>weblogic</node-manager-username>
+    <node-manager-password-encrypted>{AES256}HYCdlj4+cuu5JvY3z9bqkWNfh74sge0fXAyXbUl5gm4=</node-manager-password-encrypted>
+  </security-configuration>
+  <deployment-configuration></deployment-configuration>
+  <console-enabled>true</console-enabled>
+  <console-context-path>console</console-context-path>
+  <console-extension-directory>console-ext</console-extension-directory>
+  <server>
+    <name>admin-server</name>
+    <ssl>
+      <name>admin-server</name>
+      <listen-port>7003</listen-port>
+    </ssl>
+    <listen-address></listen-address>
+    <server-life-cycle-timeout-val>30</server-life-cycle-timeout-val>
+    <startup-timeout>0</startup-timeout>
+  </server>
+  <server-template>
+    <name>server-template_1</name>
+    <ssl>
+      <name>new_ServerTemplate_1</name>
+      <listen-port>8100</listen-port>
+    </ssl>
+    <listen-port>8001</listen-port>
+    <cluster>cluster-1</cluster>
+    <jta-migratable-target>
+      <cluster>cluster-1</cluster>
+      <migration-policy>manual</migration-policy>
+      <strict-ownership-check>false</strict-ownership-check>
+    </jta-migratable-target>
+    <server-life-cycle-timeout-val>30</server-life-cycle-timeout-val>
+    <startup-timeout>0</startup-timeout>
+  </server-template>
+  <cluster>
+    <name>cluster-1</name>
+    <cluster-messaging-mode>unicast</cluster-messaging-mode>
+    <dynamic-servers>
+      <server-template>server-template_1</server-template>
+      <maximum-dynamic-server-count>5</maximum-dynamic-server-count>
+      <calculated-listen-ports>false</calculated-listen-ports>
+      <calculated-machine-names>false</calculated-machine-names>
+      <machine-match-type>name</machine-match-type>
+      <server-name-prefix>managed-server</server-name-prefix>
+      <dynamic-cluster-size>5</dynamic-cluster-size>
+      <min-dynamic-cluster-size>1</min-dynamic-cluster-size>
+      <max-dynamic-cluster-size>8</max-dynamic-cluster-size>
+      <dynamic-cluster-cooloff-period-seconds>900</dynamic-cluster-cooloff-period-seconds>
+      <dynamic-cluster-shutdown-timeout-seconds>0</dynamic-cluster-shutdown-timeout-seconds>
+      <ignore-sessions-during-shutdown>false</ignore-sessions-during-shutdown>
+      <wait-for-all-sessions-during-shutdown>false</wait-for-all-sessions-during-shutdown>
+    </dynamic-servers>
+  </cluster>
+  <production-mode-enabled>false</production-mode-enabled>
+  <embedded-ldap>
+    <name>onprem_domain</name>
+    <credential-encrypted>{AES256}sk/lXRmHmtGWDv8KK1kp76xxwGwgg2M6Mtf73bOpixV70qXNflHXs4i4gnRhGFMs</credential-encrypted>
+  </embedded-ldap>
+  <administration-port-enabled>false</administration-port-enabled>
+  <exalogic-optimizations-enabled>false</exalogic-optimizations-enabled>
+  <config-backup-enabled>false</config-backup-enabled>
+  <configuration-version>12.2.1.4.0</configuration-version>
```

Comment:
Will it not impose any issue with WLS release other than 12.2.1.4  ?

Reply:
Good point. I have tested with config from 12.2.1.5 and run it manually and it works, but I should check if there is any constraints.

Reply:
see my response to Sankar's comment below